arcgis-ios-sdk-samples/Maps/Download preplanned map/DownloadPreplannedMapViewController.swift

```diff
@@ -0,0 +1,243 @@
+//
+// Copyright © 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import UIKit
+import ArcGIS
+
+// MARK: - Constants
+
+private enum Constants {
+    static let activityIndicatorColor = UIColor.primaryBlue
+
```

Comment:
Can you please help me better understand your expectations here?

Are you asking me to uncheck this?
![pr_feedback_01](https://user-images.githubusercontent.com/221062/63383615-fb749680-c351-11e9-9e0e-3fa57a187feb.png)

If so, are you also asking me to restore whitespace to blank lines in some way?

Reply:
> Are you asking me to uncheck this?

I am. In my experience here at Esri, most of the Objective-C/Swift code I've looked at doesn't have whitespace trimmed from whitespace-only lines.

> are you also asking me to restore whitespace to blank lines in some way?

I'm sorry for not being clear. I'm leaving the decision up to you. On one hand, it can lead to numerous whitespace changes in a diff if the code is ever updated in the future, but that may or may not ever happen. I also know (from updating other samples) that some files have had whitespace trimmed from blank lines. On the other hand, the fix is almost trivial. Select all (⌘A) and Re-Indent (⌃I) for each Swift file. I don't feel strongly either way for this PR.

Reply:
Sounds good - I'm happy to comply. Can you take a look at [d32deb3c](https://github.com/Esri/arcgis-runtime-samples-ios/pull/758/commits/d32deb3c0093d81f6680df3366b6a86f26cca3c8) and let me know if that's what you had in mind?

Reply:
Looks good! Thank you.